src/hooks/useTable.tsx

```diff
@@ -0,0 +1,54 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { useParams } from 'react-router-dom';
+
+export const useTable = () => {
+  const [people, setPeople] = useState<Person[] | null>(null);
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState(false);
```
should start with false